SharedProjects/BabylonExport.Entities/ExportParameters.cs

```diff
@@ -10,6 +10,38 @@ public enum TextureFormatExportPolicy
         CONSERVATIV, // we try to keep the source format if possible
         SIZE // we try to minimize the size of texture as much as possibe.
     }
+    public class DracoParameters
+    {
+        public const string dracoPrefix = "draco.";
+        public static readonly string compressionLevel_param_name = $"{dracoPrefix}compressionLevel";
+        public static readonly string quantizePositionBits_param_name = $"{dracoPrefix}quantizePositionBits";
+        public static readonly string quantizeNormalBits_param_name = $"{dracoPrefix}quantizeNormalBits";
+        public static readonly string quantizeTexcoordBits_param_name = $"{dracoPrefix}quantizeTexcoordBits";
+        public static readonly string quantizeColorBits_param_name = $"{dracoPrefix}quantizeColorBits";
+        public static readonly string quantizeGenericBits_param_name = $"{dracoPrefix}quantizeGenericBits";
+        public static readonly string unifiedQuantization_param_name = $"{dracoPrefix}unifiedQuantization";
+
+        public const int compressionLevel_default = 7;
```

Comment:
How were these default values chosen?

Reply:
These default values are coming from [GLTF-pipeline](https://github.com/CesiumGS/gltf-pipeline) (the Node tool we use for compression ).

Reply:
Can we add documentation that points future readers to where we got these values from?